driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTestModifications.java

```diff
@@ -30,13 +32,27 @@
 import static com.mongodb.ClusterFixture.serverVersionLessThan;
 import static com.mongodb.assertions.Assertions.assertNotNull;
 import static com.mongodb.client.unified.UnifiedTestModifications.Modifier.IGNORE_EXTRA_EVENTS;
+import static com.mongodb.client.unified.UnifiedTestModifications.Modifier.RETRY;
 import static com.mongodb.client.unified.UnifiedTestModifications.Modifier.SKIP;
 import static com.mongodb.client.unified.UnifiedTestModifications.Modifier.SLEEP_AFTER_CURSOR_CLOSE;
 import static com.mongodb.client.unified.UnifiedTestModifications.Modifier.SLEEP_AFTER_CURSOR_OPEN;
 import static com.mongodb.client.unified.UnifiedTestModifications.Modifier.WAIT_FOR_BATCH_CURSOR_CREATION;
 
 public final class UnifiedTestModifications {
-    public static void doSkips(final TestDef def) {
+    public static void applyCustomizations(final TestDef def) {
+
+        // TODO reasons for retry
+        // Exception in encryption library: ChangeCipherSpec message sequence violation
+        def.retry("TODO reason")
+                .whenFailureContains("ChangeCipherSpec message sequence violation")
+                .test("client-side-encryption", "namedKMS-createDataKey", "create datakey with named KMIP KMS provider");
+
+        def.retry("TODO reason")
+                .whenFailureContains("Number of checked out connections must match expected")
+                .test("load-balancers", "cursors are correctly pinned to connections for load-balanced clusters", "pinned connections are returned after a network error during a killCursors request");
+
+        def.retry("TODO reason")
+                .test("client-side-encryption", "namedKMS-rewrapManyDataKey", "rewrap to kmip:name1");
```

Comment:
These are flaky failures from evergreen. Reasons should be specified, or these 3 retries can be removed and added separately, after examining the failures.